manifest_staging/charts/workload-identity-webhook/values.yaml

```diff
@@ -33,3 +33,6 @@ metricsAddr: ":8095"
 metricsBackend: prometheus
 mutatingWebhookFailurePolicy: Ignore
 priorityClassName: system-cluster-critical
+objectSelector:
+  matchLabels:
+    foo: 'bar'
```

Comment:
This should be an empty value in the `values.yaml`

`mutatingWebhookObjectSelector: {}`